en/mkdocs.yml

```diff
@@ -134,6 +135,10 @@ nav:
           - MySQL: manage-databases-and-caches/choreo-managed-mysql-databases.md
           - Choreo-Managed Cache: manage-databases-and-caches/choreo-managed-caches.md
           - Billing: manage-databases-and-caches/billing-for-platform-services.md
+  - Third Party Services:
```

Comment:
How about we have a high-level section perhaps "Integrate with External Services" after the "Develop Components" section to include this content and GenAI content? 
- Integrate with External Services
      - Integrate Third Party Services
          - Overview of Third Party Services
          - Register Third Party Services
          - Manage Third Party Services
       - Integrate Generative AI Services
          - Overview of Generative AI Services
          - Register Generative AI Services
          - Manage Generative AI Services

Reply:
Since there isn't much content to have three subsections under each external service type, we can perhaps go with a single page content approach and have the sections organized properly to be accessible via the page ToC:
<img width="1543" alt="Screenshot 2024-10-04 at 10 30 21" src="https://github.com/user-attachments/assets/2e321791-f924-4808-af7a-18eb179e39ff">

Reply:
Sounds good.

Reply:
I didn't see this section in the docs. Isn't it merged yet?

Reply:
Fixed as per the suggestions https://github.com/wso2/docs-choreo-dev/pull/1587/commits/156af6e1cd4346e9e9e1010b1d3a1f829a842d74